grunt/postcss.js

```diff
@@ -1,18 +1,39 @@
-/*
+/**
  * Grunt postcss plugin
  * https://github.com/nDmitry/grunt-postcss
+ *
+ * Rules for stylelint
+ * https://github.com/stylelint/stylelint/blob/master/docs/user-guide/rules.md
  */
+
 module.exports = function( grunt, options ) {
     return {
-        autoprefixer: {
+        cssDevelopment: {
             options: {
+                map: true,
+                processors: [
+                    require( 'autoprefixer' )( {
+                        browsers: [ 'last 2 versions', 'ie >= 9', 'Firefox >= 12', 'ios >= 7', 'android >= 4' ]
+                    } ),
+                    require( 'postcss-inline-svg' ),
+                    require( 'postcss-svgo' )
+                ]
+            },
+            src: [ '<%= distPath %>css/*.css' ]
+        },
+
+        cssProduction: {
+            options: {
+                map: false,
                 processors: [
                     require( 'autoprefixer' )( {
                         browsers: [ 'last 2 versions', 'ie >= 8', 'Firefox >= 12', 'ios >= 7', 'android >= 4' ]
-                    } )
+                    } ),
+                    require( 'postcss-inline-svg' ),
+                    require( 'postcss-svgo' )
```

Comment:
Thist is new. `postcss-inline-svg` and `postcss-svgo` are not yet part of the standard set up.

Reply:
Plugins removed.